src/components/TokenClaimDialog.tsx

```diff
@@ -453,12 +453,24 @@ function useRewardClaims() {
         rewards: GaugeReward[]
       } | null,
     ) => {
-      if (!chainId || !account || !gaugeMinterContract || !library || !gauge)
+      if (!chainId || !account || !gaugeMinterContract || !library || !gauge) {
+        enqueueToast("error", "Unable to claim reward")
+        return
+      }
+
+      if (gauge.rewards.length === 0) {
+        enqueueToast("error", "No rewards to claim")
```

Comment:
do we want to enqueue an error toast if there's no rewards available? Can we just gray out the claim button instead?

Reply:
ack

Reply:
I like including this in the claim hook as well, just in case the UI doesn't disable. This should be an edge case.

Reply:
sounds good!